src/Pages/Product/components/ProductReview/ProductReview.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import './ProductReview.scss';
+
+class ProductReview extends React.Component {
+  render() {
+    const { productReviewData } = this.props;
+    const starAverage =
+      productReviewData.reduce((acc, value) => acc + value.star, 0) /
+      (productReviewData.length + 1);
+    return (
+      <div className="productReview">
+        <div className="reviewOutbox">
+          <header className="reviewTitle">
+            <h3 className="title">리뷰</h3>
+            <span className="reviewNum">
+              <span className="countReview">{productReviewData.length}</span>건
+            </span>
+          </header>
+          <section className="reviewContentsOutbox">
+            <article className="starRate">
+              <span className="starRateNum num">{starAverage}</span>
+              <div className="stars">
+                {Array(Math.ceil(starAverage))
+                  .fill(1)
+                  .map(() => {
+                    return (
+                      <img alt="별" src="/images/stars/star-on-line-sm.svg" />
+                    );
+                  })}
+                {Array(5 - Math.ceil(starAverage))
+                  .fill(1)
+                  .map(() => {
+                    return (
+                      <img alt="별" src="/images/stars/star-off-line-sm.svg" />
+                    );
+                  })}
+              </div>
+              <div className="reviewPoint">
+                <p>상품 리뷰 작성시 포인트를 드립니다</p>
+                <p>
+                  포토<strong>2,000p</strong>일반<strong>1,000p</strong>
+                </p>
+              </div>
+            </article>
+            {/* 리뷰내용 */}
+            <article>
+              <ul className="commentLists">
+                {productReviewData.map((reviewData, index) => {
+                  return (
+                    <li className="commentContent" key={reviewData.id}>
+                      <div className="commentHeading">
+                        <div className="starRaty">
+                          {Array(reviewData.star)
+                            .fill(1)
+                            .map(() => {
+                              return (
+                                <img
```

Comment:
```suggestion
                          {[...Array(reviewData.star)]
                            .map(() => <img
```
- 배열을 만들고 그 안에서 스프레드 연산자를 사용해도 이 경우에는 같은 결과를 얻을 수 있을 것 같습니다.
- map 메서드를 사용할 때 안에 화살표 함수에 중괄호를 생략할 수 있다면 가급적 그렇게 해주는 것이 가독성 측면에서 좋습니다.

Reply:
스프레드 연산자와 함수 내 중괄호, return 삭제해줬더니 훨씬 간결해졌습니다. 감사합니다 종택님~~!